### DISCUSSION TITLE
how to silence all alerts in all orgs in grafana globally.

### DISCUSSION THREAD

**Discussion by victorchen2022** [grafana/grafana#54611] [Alerting]
I migrate 8.0.6 to 9.1.2 and encounter alerts flood "no value", don't know why, and I think I need to stop annoying alerts first, don't know how to make all alerts notification silence.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
